core/logging_filters.py

```diff
@@ -0,0 +1,6 @@
+import logging
+
+
+class DebugFilter(logging.Filter):
+    def filter(self, record):
+        return (record.levelno == logging.DEBUG)
```

Comment:
This filter doesn't appear to be used anywhere, and you should consider removing or re-justifying it.